src/utils.js

```diff
@@ -6,3 +6,5 @@ export const readFile = promisify(fs.readFile);
 export const stat = promisify(fs.stat);
 export const isDir = name => stat(name).then( stats => stats.isDirectory() ).catch( () => false );
 export const isFile = name => stat(name).then( stats => stats.isFile() ).catch( () => false );
+export const stdout = console.log.bind(console); // eslint-disable-line no-console
+export const stderr = console.error.bind(console);
```

Comment:
why did you add the aliases?

Reply:
The second reason seems like a downside because you won't get that warning now with the aliases, but that was also the case with the current setup

Reply:
Yep, totally agreed! More or less nobody is stopping you from just using `stdout.write` if you want to avoid eslint screaming at you 😸From my point of view, this eslint rule is about _how_ you approach printing something to the terminal. Having exception just in one place doesn't suggest to other developers that are working on the project that you can just throw around `console.log` statements all over the place, compared to having `consle.log` everywhere and using `eslint-ignore`, but this is partially true for using `stdout(err)` also.

And, again, this is my highly opinionated thoughts on the topic. If you feel that this library is better off without these aliases, I don't mind removing them 😄